Gemfile

```diff
@@ -40,6 +40,8 @@ group :development, :test do
 
   # Testing framework
   gem "rspec-rails", "~> 4.0.0.beta3"
+  # Adds support for Capybara system testing and selenium driver
```

Comment:
what's the thinking behind moving `capybara` to both the `:development` and `:test` groups, as opposed to just keeping it in `:test`?

Reply:
I'm just used to seeing Rspec and capybara in the same group. Happy to move it back if you would prefer.

Reply:
I don’t think it really matters much either way, I just wanted to make sure I’m not missing anything.